### ISSUE TITLE
Compiling issues on fresh cloned TempoSample and Templo plugin

### ISSUE THREAD

**Issue by Blyron** [tempo-sim/Tempo#153]
**Description**
A clear and concise description of what the bug is and the context in which you encountered it.
Please include:
 - Severity (blocked/slowed/inconvenienced): Blocked
 - Reproduces (always/sometimes/rarely): Always
 - Platform (Windows/Mac/Linux): Linux 
 - Tempo version (commit or release): Latest
 - Local changes (branch): main
 - Stack trace:
[LogsTempo.txt](https://github.com/user-attachments/files/17799819/LogsTempo.txt)

**To Reproduce**
If possible, include steps to reproduce.

Run; Scripts/Build.sh

**Screenshots**
If applicable, add screenshots or files to help explain the bug.

**Additional context**

**Comment by pete-tempo** [comment]
Hi @Blyron can you confirm if you have run the `Setup.sh` script in Tempo? Take a look at the [Getting Started](https://github.com/tempo-sim/Tempo#getting-started) section of the readme if you haven't.  We should make this error clearer.

**Comment by Blyron** [comment]
Yeah. I have run it. And it still appears. I will do it again and I will show the logs tomorrow

**Comment by Blyron** [comment]
Hello I am back, it seems like Setup.sh is failling due to jq 
`
cvc-246@cvc-246:~$ cd tempo/
cvc-246@cvc-246:~/tempo$ ls
Config   Intermediate  Plugins    Scripts  TempoSample.uproject
Content  LICENSE       README.md  Source
cvc-246@cvc-246:~/tempo$ Scripts/
Build.sh    Clean.sh    Package.sh  Rename.sh   Run.sh      
cvc-246@cvc-246:~/tempo$ Scripts/
Build.sh    Clean.sh    Package.sh  Rename.sh   Run.sh      
cvc-246@cvc-246:~/tempo$ cd Plugins/Tempo/
cvc-246@cvc-246:~/tempo/Plugins/Tempo$ ls
EngineMods  README.md  TempoAgents      TempoMovement   TempoSensors
External    Scripts    TempoCore        TempoROS        TempoWorld
LICENSE     Setup.sh   TempoGeographic  TempoROSBridge
cvc-246@cvc-246:~/tempo/Plugins/Tempo$ ./Setup.sh 

Installing Tempo Engine Mods

jq: error (at /home/cvc-246/tempo/Plugins/Tempo/EngineMods/EngineMods.json:51): Cannot iterate over null (null)
Checking ThirdParty dependencies...

Expected third party dependency hash 01fbcee1ab08451bc0326ae9331d4210 but found ed962a5df01e2eb28e765a4727d9bb17 in /home/cvc-246/tempo/Plugins/Tempo/TempoCore/Source/ThirdParty/gRPC. Update? (y/N): y

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 75413    0 75413    0     0   135k      0 --:--:-- --:--:-- --:--:--  135k

Downloading TempoThirdParty release v0.6 for platform Linux from https://api.github.com/repos/tempo-sim/TempoThirdParty/releases/assets/169733433

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  144M  100  144M    0     0  21.9M      0  0:00:06  0:00:06 --:--:-- 24.8M

Successfully downloaded archive to /tmp/tmp.dLKVWROw2E/TempoThirdParty-gRPC-Linux-v0.6.tar.gz

Extracting archive to /home/cvc-246/tempo/Plugins/Tempo/TempoCore/Source/ThirdParty

New hash: ed962a5df01e2eb28e765a4727d9bb17

Checking ThirdParty dependencies...

Skipping check of TempoROS ThirdParty dependencies because TempoROS plugin is not enabled in TempoSample.uproject
cvc-246@cvc-246:~/tempo/Plugins/Tempo$ jq
jq - commandline JSON processor [version 1.6]

Usage:	jq [options] <jq filter> [file...]
	jq [options] --args <jq filter> [strings...]
	jq [options] --jsonargs <jq filter> [JSON_TEXTS...]

jq is a tool for processing JSON inputs, applying the given filter to
its JSON text inputs and producing the filter's results as JSON on
standard output.

The simplest filter is ., which copies jq's input to its output
unmodified (except for formatting, but note that IEEE754 is used
for number representation internally, with all that that implies).

For more advanced filters see the jq(1) manpage ("man jq")
and/or https://stedolan.github.io/jq

Example:

	$ echo '{"foo": 0}' | jq .
	{
		"foo": 0
	}

For a listing of options, use jq --help.

`

**Comment by pete-tempo** [comment]
Thanks @Blyron this is helpful. Could you:
- Confirm you have UE 5.4.4 installed from [here](https://www.unrealengine.com/en-US/linux)? We do not support 5.5 yet.
- If so, please confirm if this file exists? `$UNREAL_ENGINE_PATH/Engine/Intermediate/Build/BuildRules/UE5RulesManifest.json`
- And if it does, please reply with the output of this command? `jq -r '.EngineVersion' "$UNREAL_ENGINE_PATH/Engine/Intermediate/Build/BuildRules/UE5RulesManifest.json"`

Also, I recommend upgrading to the new Tempo release 0.2 (just do a `git pull --recurse-submodules` from the release branch).

**Comment by pete-tempo** [comment]
The latest Tempo release includes support for Unreal 5.5 and a clearer error for unsupported versions.
